### ISSUE TITLE
Objective-C Compatible?

### ISSUE THREAD

**Issue by mobileonekh** [CaliCastle/PopMenu#30]
Hi,
Does this control compatible with Objective-C?

Thanks;

**Comment by CaliCastle** [comment]
Hi, as far as I know this is compatible with Objective-C, let me know if it doesn't, thanks!

**Comment by kmyhy** [comment]
No, this control didn't support Objective-C project. The swift-oc-bridge file which is auto-generated by Xcode didn't expose any available method.

**Comment by dezinezync** [comment]
The bridge file generated by Xcode has the following: 

```objc
/// The object responsible for managing a pop menu <code>PopMenuViewController</code>
SWIFT_CLASS("_TtC7PopMenu14PopMenuManager")
@interface PopMenuManager : NSObject
- (nonnull instancetype)init SWIFT_UNAVAILABLE;
+ (nonnull instancetype)new SWIFT_DEPRECATED_MSG("-init is unavailable");
@end
```

As noted by @kmyhy, the methods are not exposed.

**Comment by dezinezync** [comment]
I was able to do some testing on the matter, and unfortunately, given how things are configured PopMenu won't be Obj-C compatible without some critical refactoring. 

PopMenu uses Swift structs in quite a few places which isn't accessible/visible to Obj-C projects. I'm hoping to make some time and go through this more thoroughly to see if something else is feasible.
